src/pages/blogs/ios-stable.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: ../../layouts/MarkdownLayout.astro
+title: Blog | iOS Stable
+description: Shorebird Code Push for iOS stable announcement
+---
+
+# Announcing Code Push iOS 1.0 🥳
+
+Shorebird Code Push allows you to update your Flutter app instantly over the air
```

Comment:
@eseidel did you already write something up? Otherwise I can try to do an initial pass of updates.